src/transform-sdk/js/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++ -fno-exceptions")
 if(Redpanda_ENABLE_SANITIZERS)
   add_link_options(-fsanitize=address,leak,undefined)
   add_compile_options(-fsanitize=address,leak,undefined)
+  set(CONFIG_ASAN ON)
+  set(CONFIG_UBSAN ON)
```

Comment:
nitpick: is it preferable to group the sanitizer configs with the diffs that fix the errors?

Reply:
This doesn't actually work for me locally (which is why I added the Bazel build). I'll revert

Reply:
oh bummer :(